<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 主窗口渐变背景 &amp; 基本文字 */
    QWidget#SignUp {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #d0f0ff,
            stop:1 #ffffff
        );
        font-family: &quot;Microsoft YaHei&quot;;
    }

    /* 标题 */
    #label_title {
        color: #1e88e5;
        font-size: 26px;   /* 放大字体 */
        font-weight: bold; /* 加粗 */
    }

    /* 提示标签（保留原名tip1Label/tip2Label） */
    QLabel {
        color: #0f2b4a;
    }
    #tip1Label, #tip2Label {
        color: #1e88e5;
        font-weight: 600;
    }

    /* 输入框样式（与登录页一致） */
    QLineEdit {
        border: 2px solid #81d4fa;
        border-radius: 12px;
        padding: 8px 10px;
        background-color: #ffffff;
        color: #0f1f38;
    }
    QLineEdit:focus {
        border: 2px solid #1e88e5;
    }

    /* 单选按钮颜色 */
    QRadioButton {
        color: #1e88e5;
        spacing: 6px;
    }

    /* 按钮样式（与登录页一致） */
    QPushButton {
        border-radius: 12px;
        border: 1px solid #64b5f6;
        color: #ffffff;
        font-weight: bold;
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #6ec6ff,
            stop:1 #1e88e5
        );
        padding: 8px 16px;
    }
    QPushButton:hover {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #81d4fa,
            stop:1 #1976d2
        );
    }
    QPushButton:pressed {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #1e88e5,
            stop:1 #1565c0
        );
    }

    /* 小控件的最小高度保证显示（保留原有像素感） */
    QLineEdit {
        min-height: 36px;
    }
    QPushButton {
        min-height: 40px;
        min-width: 90px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>用户注册</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="requiredVBox">
     <item>
      <widget class="QLabel" name="tip1Label">
       <property name="text">
        <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;必填信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_required">
       <item row="0" column="0">
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="usernameLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_usertype">
         <property name="text">
          <string>用户类型：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="hLayout_userType">
         <item>
          <widget class="QRadioButton" name="doctorRadioButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>医生</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="patientRadioButton">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>患者</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="optionalVBox">
     <item>
      <widget class="QLabel" name="tip2Label">
       <property name="text">
        <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;选填信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_optional">
       <item row="0" column="0">
        <widget class="QLabel" name="label_gender">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="genderLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_age">
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ageLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>年龄</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_phone">
         <property name="text">
          <string>电话：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="phoneNumberLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>电话</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_email">
         <property name="text">
          <string>邮箱：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="emailLineEdit">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>56</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_buttons">
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="signupButton">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>58</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="minimumSize">
        <size>
         <width>124</width>
         <height>58</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
